pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -216,9 +226,23 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 		o.CurrentReports = append(o.CurrentReports, report)
 		allExecutionIDs = append(allExecutionIDs, weid.WorkflowExecutionId)
 
+		if seenWorkflows[weid.WorkflowId] >= (r.config.F + 1) {
+			// Update the last seen round for this outcome. But this should only happen if the workflow is seen by F+1 nodes
+			outcome.LastSeenAt = outctx.SeqNr
+		}
+
 		o.Outcomes[weid.WorkflowId] = outcome
 	}
 
+	// We need to prune outcomes from previous workflows that are no longer relevant.
+	for workflowID, outcome := range o.Outcomes {
+		// TODO: 3,600 is the amount of rounds we allow as threshold. This should be configurable.
```

Comment:
We should document somewhere that we are making the assumption that 1 round == 1 sec

Reply:
agreed

Reply:
I'd rather simply state that this is affected by OCR round time, without referencing the 1sec value.